config/prod/xschildw-shinyproxy.yaml

```diff
@@ -40,9 +40,7 @@ parameters:
 
 # For CI system (do not change)
 hooks:
-  before_create:
-    - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/aws-infra/master/managed-ec2-linux-v1.j2 --create-dirs -o templates/remote/managed-ec2-linux-v1.j2"
-  before_update:
+  before_launch:
     - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/aws-infra/master/managed-ec2-linux-v1.j2 --create-dirs -o templates/remote/managed-ec2-linux-v1.j2"
```

Comment:
@zaro0508 I believe this is the file that you said I had missed in your review, but it looks updated to me. Is there something I'm missing?